ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -1123,7 +1126,22 @@ func (pg *DEXOnboarding) postBond() {
 	}
 
 	pg.isLoading = true
-	// Request for wallet password before attempting to post bond.
+	walletID, err := pg.dexc.WalletIDForAsset(bondAsset.ID)
+	if err != nil {
+		pg.notifyError(err.Error())
+		return
+	}
+
+	if walletID != nil && pg.AssetsManager.WalletWithID(*walletID) != nil {
```

Comment:
If `walletID != nil` but `WalletWithID` is nil, what happens? Will core accept adding a wallet for an asset that already has a wallet?

Reply:
> Will core accept adding a wallet for an asset that already has a wallet?

Errm, it won't

Reply:
updated